sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -553,9 +553,7 @@
             <excludePackageNames>
               *.impl*:
               *.implementation*:
-              <!-- RMDocs do utils and samples actually do anything -->
-              *.utils:
```

Comment:
will anyone get affected on removed `*.utils:`?

Reply:
No, the utils and samples are added when mgmt pom migrate.